boot/cabal_macros.template.h

```diff
@@ -28,12 +28,12 @@
 
 {% if notNull packageKey %}
 #ifndef CURRENT_PACKAGE_KEY
-#define CURRENT_PACKAGE_KEY "{{ packageKey }}"
+#define CURRENT_PACKAGE_KEY "{{ mkBuilder packageKey }}"
```

Comment:
I won't accept this. This makes template harder to read/edit for the benefit of non-noticeable speedup (microseconds! not even milliseconds).